### ISSUE TITLE
Project website inaccessible

### ISSUE THREAD

**Issue by almereyda** [researchstudio-sat/webofneeds#3182]
When trying to access any link to `sat.researchstudio.at`, I am always redirected to `www.researchstudio.at/`. Hence most of the referenced links at the bottom of the README are inaccessible.

**Comment by fkleedorfer** [comment]
Yes, our organization is getting a new website and it is in an intermediate state. Until I can fis those links, you can access most of the publications on my researchgate profile. Can't give you the link currently because it seems researchgate is having problems, but you'll find it once rg comes back up.
